regex-syntax/src/parser.rs

```diff
@@ -185,6 +185,16 @@ impl Parser {
             return Ok(try!(self.lit(c)));
         }
         match c {
+            '\n' => { self.bump();
+                while self.cur().is_whitespace() { self.bump(); };
+                Ok(Build::Expr(Expr::Empty))
+            }
+            '\r' => { self.bump();
+                    if self.bump_if('\n') {
+                        while self.cur().is_whitespace() { self.bump(); };
+                        Ok(Build::Expr(Expr::Empty))
+                    }
+                    else { Err(self.err(ErrorKind::UnrecognizedEscape(c))) } },
```

Comment:
Could you please format this code so that it's consistent with the rest of the code in this library? e.g., Consistent indentation, new lines after braces, etc.